win_build_vs.bat

```diff
@@ -73,12 +73,18 @@ for %%x in (%*) do (
     if [%%~x] EQU [/NONFREEUCRT] set "redist=-DMSI_REDISTRIBUTE_UCRT_NONASL=ON"
     if [%%~x] EQU [/L]           set build_linter=ON
     if [%%~x] EQU [/RO]          set real_odbc=ON
+    if [%%~x] EQU [/NJ]          set generator="Ninja"
```

Comment:
I would prefer `/NINJA`

Reply:
I think it's better to be more explicit, updated in 5aa6d9a5942b642f49b0903ee80d1f74fc0ec82c